test-requirements.txt

```diff
@@ -0,0 +1,11 @@
+bumpversion==0.5.3
```

Comment:
Doesn't this make sense to be in tests directory ?

Reply:
Well, this isn't just test dependencies. It is more like development dependencies. The tests directory only contains tests